tools/changelogger/src/render.rs

```diff
@@ -232,9 +235,19 @@ fn render_entry(entry: &HandAuthoredEntry, mut out: &mut String) {
     if !maintainers().contains(&entry.author.to_ascii_lowercase().as_str()) {
         references.push(format!("@{}", entry.author.to_ascii_lowercase()));
     };
+    meta.push('(');
+    let sep = if include_affectee {
+        write!(meta, "{}", entry.meta.target.unwrap_or_default()).unwrap();
+        ", "
+    }
+    else {
+        ""
+    };
     if !references.is_empty() {
-        write!(meta, "({}) ", references.join(", ")).unwrap();
+        write!(meta, "{sep}{}", references.join(", ")).unwrap();
     }
+    meta.push_str(") ");
```

Comment:
I think this can be simplified by just doing `references.insert(0, entry.meta.target)` so that you don't have to do manual comma delimitation.